src/plugins/vis_type_timeseries/server/routes/vis.ts

```diff
@@ -41,6 +42,7 @@ export const visDataRoutes = (
     },
     async (requestContext, request, response) => {
       try {
+        validateObject(request.body);
```

Comment:
@flash1293 could you please confirm that it's exactly is what you described in #78908. 

Honestly I don't fully understand why we need it cause I've tried to add _`_proto__, prototype`  properties into payload and got the following error: `"Invalid request payload JSON format"`
Just want to be double sure that we really need it